src/seedu/addressbook/AddressBook.java

```diff
@@ -315,34 +319,53 @@ private static boolean isValidFilePath(String filePath) {
         if (filePath == null) {
             return false;
         }
-        Path filePathToValidate;
+        return isValid(filePath);
+    }
+
+	private static boolean isValid(String filePath) {
```

Comment:
isValid is a very vague method name. Are you sure `isValidFilePath` is not adequate?